src/Repository/ElectedRepresentative/ElectedRepresentativeRepository.php

```diff
@@ -118,12 +132,23 @@ public function hasActiveMandate(Adherent $adherent): bool
         ;
     }
 
-    private function createFilterQueryBuilder(ListFilter $filter): QueryBuilder
+    private function createFilterQueryBuilder(ListFilter $filter, bool $forJME = false): QueryBuilder
     {
         $qb = $this
             ->createQueryBuilder('er')
         ;
 
+        if ($forJME) {
+            $qb
+                ->andWhere(
+                    sprintf(
+                        'er.adherent IS NOT NULL%s',
+                        $filter->getCreatedByAdherent() ? ' OR er.createdByAdherent = '.$filter->getCreatedByAdherent()->getId() : ''
+                    )
+                )
+            ;
```

Comment:
1/ this sprintf to build conditions looks weird
2/ let's use query parameters

what about:

```
$adherentCondition = new Doctrine\ORM\Query\Expr\Orx();

$adherentCondition->add('er.adherent IS NOT NULL');

if ($filter->getCreatedByAdherent()) {
    $adherentCondition->add('er.createdByAdherent = :created_by_adherent');
    $qb->setParameter('created_by_adherent', $filter->getCreatedByAdherent());
}

$qb->andWhere($adherentCondition);
```